### ISSUE TITLE
Field size added to field name

### ISSUE THREAD

**Issue by ondras** [ondras/wwwsqldesigner#149]
_From [Shylux](https://code.google.com/u/Shylux/) on December 12, 2011 09:38:51_

I created a single table, no relations just some fields.
Now every time i close a field details section the size is added to the field name.

This results in field names like "email (100) (100) (100)".

I use firefox 8.0.1 and Ubuntu 10.04.

_Original issue: http://code.google.com/p/wwwsqldesigner/issues/detail?id=142_

**Comment by ondras** [comment]
_From [Shylux](https://code.google.com/u/Shylux/) on December 12, 2011 02:03:58_

Now it works perfectly..
The issue appears after i export to xml and then import it.

**Comment by ondras** [comment]
_From [ondrej.zara](https://code.google.com/u/ondrej.zara/) on December 12, 2011 03:22:06_

Are you trying the hosted demo version ( http://ondras.zarovi.cz/sql/demo/ ), or a downloaded release (2.6) ?

I believe I already fixed this, but only after 2.6 was released...

**Comment by ondras** [comment]
_From [Shylux](https://code.google.com/u/Shylux/) on December 12, 2011 03:46:01_

I downloaded release 2.6.

**Comment by ondras** [comment]
_From [ondrej.zara](https://code.google.com/u/ondrej.zara/) on December 13, 2011 03:54:10_

Okay then; this is already fixed in latest SVN revision.

**Status:** Duplicate

**Comment by ondras** [comment]
_From [ondrej.zara](https://code.google.com/u/ondrej.zara/) on February 28, 2012 07:18:09_

Issue 149 has been merged into this issue.

**Comment by ondras** [comment]
_From [ondrej.zara](https://code.google.com/u/ondrej.zara/) on May 20, 2012 12:21:05_

Issue 153 has been merged into this issue.

**Comment by ondras** [comment]
_From [ondrej.zara](https://code.google.com/u/ondrej.zara/) on June 05, 2012 11:41:07_

Issue 154 has been merged into this issue.
